### PR TITLE
Ensure appeals are not showing up twice for Reader and Queue users

### PR DESCRIPTION
**Pull request by aroltsch**
connects #4476 

Tested that the query works for both Reader and Queue queue. Will test the final result by running the query from prod rails console:

```result = VACOLS::CaseAssignment.tasks_for_user("bvajfreeman")```

as of right now there are duplicate appeals:

```
result.map(&:vacols_id)
=> ["1900552", "3593491", "3593491", "3619343"]
```

### CODE REVIEW

File: app/models/vacols/case_assignment.rb [hunk 1]
```diff
@@ -56,16 +59,12 @@ def select_tasks
...
              "folder.tinum as docket_number")
-        .joins(<<-SQL)
-          LEFT JOIN decass
-            ON brieff.bfkey = decass.defolder
-          JOIN staff
-            ON brieff.bfcurloc = staff.slogid
-          JOIN folder
-            ON brieff.bfkey = folder.ticknum
-        SQL
+        .joins("left join vacols.decass on brieff.bfkey = decass.defolder")
```
Comment (amprokop) [inline app/models/vacols/case_assignment.rb]: any reason for this refactor?

File: app/repositories/queue_repository.rb [hunk 2]
```diff
@@ -35,7 +35,7 @@ def self.appeals_from_tasks(tasks)
 
   # :nocov:
   def self.tasks_query(css_id)
-    VACOLS::CaseAssignment.active_cases_for_user(css_id)
+    VACOLS::CaseAssignment.tasks_for_user(css_id)
```
Comment (amprokop) [inline app/repositories/queue_repository.rb]: thanks 🙏

File: app/models/vacols/case_assignment.rb (hunk 1 above)
Comment (aroltsch) [reply app/models/vacols/case_assignment.rb]: I don't even know why I changed it. I thought I liked it this way better. lol

Comment (amprokop) [reply app/models/vacols/case_assignment.rb]: hmm ok. just flagging that @mdbenjam mentioned he couldn't find a way to do a left join. this looks like it should work — are we sure it functions identically?

Comment (aroltsch) [reply app/models/vacols/case_assignment.rb]: I am pretty sure it works.

Comment (mdbenjam) [reply app/models/vacols/case_assignment.rb]: It's basically the exact same thing right? Just separating the joins and not capitalizing 'left', 'join', and 'on' :)

**Pull request merged**